introspection-engine/connectors/sql-introspection-connector/tests/db_specific_introspection/postgres/tables_postgres.rs

```diff
@@ -229,3 +229,51 @@ async fn introspecting_a_table_with_a_multi_column_non_unique_index_should_work(
     let result = dbg!(api.introspect().await);
     custom_assert(&result, dm);
 }
+
+#[test_one_connector(connector = "postgres")]
+async fn introspecting_a_table_enums_should_work(api: &TestApi) {
+    let sql = format!("CREATE Type Color as ENUM ( 'black', 'white')");
+    let sql2 = format!("CREATE Type Color2 as ENUM ( 'black2', 'white2')");
```

Comment:
I'm pretty sure the implementation is correct, but I'd also test creating an enum with the variants in non-alphabetical order to check that we still get them sorted by alphabetical order, but it's very nitpicky